core/arch/arm/plat-imx6/imx6_init.S

```diff
@@ -311,15 +311,15 @@ loop_1:
 	bne loop_1
 
 	/* Allow NSec to manage FIQ/Imprecise abort */
-	mrc p15, 0, r0, c1, c1, 0    /* read Secure Configuration Register */
-	orr r0, r0, #0x30            /* SCR[FW]=1, SCR[AW]=1 */
-	mcr p15, 0, r0, c1, c1, 0    /* write Secure Configuration Register */
+	read_scr r0    		/* read Secure Configuration Register */
+	orr r0, r0, #0x20  	/* SCR[FW]=0, SCR[AW]=1 */
```

Comment:
The comment isn't accurate as `SCR_FW` isn't cleared. Please use `SCR_AW` instead of `#0x20`.